sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -19,6 +19,7 @@ The Feature Test Macro SYCL\_EXT\_INTEL\_DEVICE\_INFO will be defined as one of
 | 4     | Free device memory query is supported |
 | 5     | Device ID is supported |
 | 6     | Memory clock rate and bus width queries are supported |
+| 7     | Throttle reasons, fan speed and power limits queries are supported |
```

Comment:
It looks like the implementation still defines this macro as `6`.

Reply:
@againull do we support the 7th revision?

Reply:
Yes, it seems like this was a bug in the release branch for 6_2, but I think it is too late to fix for that release.  Can you create a separate PR against the main "sycl" branch that fixes the macro definition?